src/components/Navbar.js

```diff
@@ -13,29 +32,91 @@ const Navbar = () => {
         <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
           Music
         </Typography>
-        {/*when user is loggedin can not see login and signup buttons  */}
-        {!isLoggedIn && (
-          <>
-            <Button color="inherit" component={Link} to="/signIn">
-              Login
-            </Button>
-            <Button color="inherit" component={Link} to="/register">
-              Signup
-            </Button>
-          </>
-        )}
-        <Button color="inherit" component={Link} to="/home">
-          Home
-        </Button>
-        {/*only if user is logged in can see the logout button  */}
-        {isLoggedIn && (
-          <>
-            <Button color="inherit" component={Link} to="/profile">
-              Profile
-            </Button>
-            <Logout />
-          </>
-        )}
+        <Box sx={{ display: { xs: 'none', sm: 'block' } }}>
+          {/*when user is loggedin can not see login and signup buttons  */}
+          {!isLoggedIn && (
+            <>
+              <Button color="inherit" component={Link} to="/signIn">
+                Login
+              </Button>
+              <Button color="inherit" component={Link} to="/register">
+                Signup
+              </Button>
+            </>
+          )}
+          <Button color="inherit" component={Link} to="/home">
+            Home
+          </Button>
+          {/*only if user is logged in can see the logout button  */}
+          {isLoggedIn && (
+            <>
+              <Button color="inherit" component={Link} to="/profile">
+                Profile
+              </Button>
+              <Logout />
+            </>
+          )}
+        </Box>
+        <Box sx={{ display: { xs: 'block', sm: 'none' } }}>
+          <IconButton
+            size="large"
+            aria-label="account of current user"
+            aria-controls="menu-appbar"
+            aria-haspopup="true"
+            onClick={() => setAnchorElUser(true)}
```

Comment:
I'm reading the material-ui documentation and it looks like `Menu` component expects `anchorEl` to be an HTML Element or a function that returns an HTML element: [https://mui.com/material-ui/api/menu/#Menu-prop-anchorEl](https://mui.com/material-ui/api/menu/#Menu-prop-anchorEl)

But here you're setting it to a boolean, which is resulting in some errors that say `"expected one of type [function]"`:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/eca3d229-c7f5-40c4-969c-4500e8414429)

If we look at the example for positioned menu here in the docs: [https://mui.com/material-ui/react-menu/#positioned-menu](https://mui.com/material-ui/react-menu/#positioned-menu)

We can see that they use the `onClick` handler and set the "anchorEl" value to the button that was clicked. This tells the `Menu` component where to anchor/position the menu that is displayed:

```suggestion
            onClick={event => setAnchorElUser(event.currentTarget)}
```